### ISSUE TITLE
Remove gridstack dependency

### ISSUE THREAD

**Issue by martinRenou** [QuantStack/voila#204]
It feels bad to have `voila-gridstack` as a test dependency. We should provide a `test-template` or something equivalent.

**Comment by SylvainCorlay** [comment]
Absolutely! Although we should find a means to not install the test template by default...

**Comment by martinRenou** [comment]
Yeah, one idea would be have a test command that:
- install the test template
- run the tests
- uninstall the test template

**Comment by SylvainCorlay** [comment]
Sounds good. I also wondered if the [test] feature in setup.py could include the test template.

**Comment by maartenbreddels** [comment]
I think if we should a template in the `tests\test_template` directory, and make the test pick that up using cmd line argument, and on top of that we can put a setup.py in it to make it install it as well, so we test both cases.

**Comment by martinRenou** [comment]
That is basically what I wanted to do :)
